lib/src/app.dart

```diff
@@ -16,7 +17,10 @@ class MyApp extends StatelessWidget {
       providers: [
         BlocProvider(
           create: (context) => SignUpBloc(),
-        )
+        ),
+        BlocProvider(
+          create: (context) => LoginBloc(),
+        ),
```

Comment:
Restrict the use of `Bloc Provider` on the app scope. It is recommended to use `Bloc Provider` on the screen that uses this bloc